src/libstd/collections/hash/map.rs

```diff
@@ -10,6 +10,8 @@
 //
 // ignore-lexer-test FIXME #15883
 
+#![stable]
```

Comment:
Well, we create a new module and pub use * in them, I'm not sure if that counts? e.g: https://github.com/rust-lang/rust/blob/master/src/libcollections/lib.rs#L77